packages/components/psammead-image/src/testHelpers/stories.jsx

```diff
@@ -69,5 +69,20 @@ const stories = (
       ),
       { notes },
     );
+  if (Component.name === 'Img') {
```

Comment:
Why are you doing this `if`?

Reply:
This is done to avoid creating a story for `AmpImg`, since testHelpers is used to create both `Img` and `AmpImg` stories.

Reply:
I dont think we need it, it didnt need it when `height` was made optional. Just dont pass `width`

Reply:
I could remove the `if` but that means we create a story of `AmpImg` without `width`. `width` being a required prop on `AmpImg`, it doesn't make sense to have a story to preview the behaviour of `AmpImg` when `width` is not provided.

Reply:
Why didnt we need one for height?

Reply:
If you want a seperate one, can you make it a separate export that we can just use for the normal image? Reusable tests shouldnt have specific edge case handling